src/main/java/com/trailfinder/dto/ProfileDTO.java

```diff
@@ -8,6 +8,7 @@
  */
 public class ProfileDTO {
 
+	//private fields
```

Comment:
This comment is not needed.  One can tell they are private by looking at the access modifier.